lib/synced/strategies/full.rb

```diff
@@ -192,6 +192,15 @@ def search_params
         end]
       end
 
+      def scope_params
+        case @scope
+        when Rental
```

Comment:
By default models would return empty hash here, but when including the module it would do some meta magic, for Rental it would return  `{  rental_id: self.synced_id }` etc.